src/style.css

```diff
@@ -1,3 +1,90 @@
-body {
+html {
+  font-family: Roboto, sans-serif;
+}
+
+ul,
+ol {
+  padding: 0;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+p,
+ul,
+ol,
+li,
+figure,
+figcaption,
+blockquote,
+dl,
+dd {
   margin: 0;
+  list-style: none;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__item {
+  margin-right: 20px;
```

Comment:
You should be consistent with your vertical margins. This is a code quality issue because inconsistent vertical margins can lead to layout issues and make the CSS harder to maintain. It's better to add only top or only bottom margin, but not both.